rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] I would recommend adding `book` and `person` to the attribute accessor as well as you need the getter and setter both.

Reply:
Thank you for this, @Dipeshtwis.
 I have effected the change.
Cheers.